components/authz-service/storage/v2/memstore/memstore.go

```diff
@@ -369,7 +369,7 @@ func (*State) ApplyStagedRules(context.Context) error {
 	return nil
 }
 
-func (*State) FindMissingProjects(ctx context.Context, projectIDs []string) ([]string, error) {
+func (*State) MapAllAppliedRulesToProjects(ctx context.Context) (map[string][]*storage.Rule, error) {
 	// TODO
```

Comment:
Is this really a TODO? If so, can we elaborate a bit? If not, remove the comment, please? 😃

Reply:
Yeah sorry that was thoughtless copy pasta.